llama-index-integrations/readers/llama-index-readers-zyte-serp/llama_index/readers/zyte_serp/base.py

```diff
@@ -0,0 +1,82 @@
+from typing import Literal, Optional
+from pydantic import Field
+
+from llama_index.core.readers.base import BasePydanticReader
+from llama_index.core.schema import Document
+
+
+class ZyteSerpReader(BasePydanticReader):
+    """Get google search results URLs for a search query.
+
+    Args:
+        api_key: Zyte API key.
+        extract_from: Determines the mode while extracting the .
+            It can take one of the following values: 'html', 'html-text', 'article'
```

Comment:
I think this should be httpResponseBody or browserHtml

Reply:
Done ;ere: https://github.com/rakeshmehta0308/llama_index/pull/2/commits/63ecfd4e4e437e007edc94d5bca05fa38b6a354a